### PR TITLE
UIREQ-775: Create/Edit Request: required is not read

### PR DESCRIPTION
**Pull request by Dmitriy-Litvinenko**
## Purpose
Create/Edit Request: required is not read

## Approach
For resolve this problem we add minimal changes in UI that was approved by PO
UIREQ-775

## Refs
https://issues.folio.org/browse/UIREQ-775

### CODE REVIEW

**Review by artem-blazhko — APPROVED** [review]

**Review by pavelspichonak — APPROVED** [review]

**Review by artem-blazhko — APPROVED** [review]

**Review by mkuklis — APPROVED** [review]

File: src/RequestForm.js [hunk 1]
```diff
@@ -1374,26 +1366,30 @@ class RequestForm extends React.Component {
                                         name="item.barcode"
                                         placeholder={placeholder}
                                         aria-label={<FormattedMessage id="ui-requests.item.barcode" />}
+                                        label={<FormattedMessage id="ui-requests.item.barcode" />}
```
Comment (zburke) [inline src/RequestForm.js:1369]: Please omit `aria-label` now that this field properly has `label`.

File: src/RequestForm.js [hunk 2]
```diff
@@ -1296,25 +1288,29 @@ class RequestForm extends React.Component {
                                           name="instance.hrid"
                                           placeholder={placeholder}
                                           aria-label={<FormattedMessage id="ui-requests.instance.value" />}
+                                          label={<FormattedMessage id="ui-requests.instance.value" />}
```
Comment (zburke) [inline src/RequestForm.js:1291]: Please omit `aria-label` now that this field properly has a `label`.

File: src/RequestForm.js [hunk 3]
```diff
@@ -1586,12 +1578,14 @@ class RequestForm extends React.Component {
                                     name="requester.barcode"
                                     placeholder={placeholder}
                                     aria-label={<FormattedMessage id="ui-requests.requester.barcode" />}
+                                    label={<FormattedMessage id="ui-requests.requester.barcode" />}
```
Comment (zburke) [inline src/RequestForm.js:1581]: Please omit `aria-label` now that this field properly has a `label`.

**Review by zburke — CHANGES_REQUESTED** [review]
Fields should have either `label` or `aria-label`, but not both.

File: src/RequestForm.js (hunk 1 above)
Comment (Dmitriy-Litvinenko) [reply src/RequestForm.js]: Fixed

File: src/RequestForm.js (hunk 3 above)
Comment (Dmitriy-Litvinenko) [reply src/RequestForm.js]: Fixed

**Pull request merged**